_data/projectfooter.yml

```diff
@@ -7,6 +7,8 @@ links:
         url: /downloads
       - page: Docs
         url: https://docs.wildfly.org
+      - page: Get Started
+        url: /getstarted
```

Comment:
Just wondering if the URL should be /get-started instead of /getstarted. Took a quick look at quarkus.io and they are using /get-started.

Reply:
I think `/get-started` is better. I'll fix that...